package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -551,46 +622,57 @@ def run(self, **kwargs):
                 self._update_water_selection()
 
             s1_frame_results_dict = {}
-            if self.selection1_type in ('donor', 'both') and self._water_acceptors:
+            if self.selection1_type in ('donor', 'both') and \
+            self._water_acceptors:
                 self.logger_debug("Selection 1 Donors <-> Water Acceptors")
                 ns_acceptors = AtomNeighborSearch(self._water_acceptors)
                 for i, donor_h_set in self._s1_donors_h.items():
                     d = self._s1_donors[i]
                     for h in donor_h_set:
                         res = ns_acceptors.search(h, self.distance)
                         for a in res:
-                            donor_atom = h if self.distance_type != 'heavy' else d
+                            donor_atom = h if self.distance_type != 'heavy' \
+                            else d
                             dist = self.calc_eucl_distance(donor_atom, a)
                             if dist <= self.distance:
                                 angle = self.calc_angle(d, h, a)
                                 if angle >= self.angle:
                                     self.logger_debug(
                                         "S1-D: {0!s} <-> W-A: {1!s} {2:f} A, {3:f} DEG".format(h.index, a.index, dist, angle))
-                                    if not (a.resname, a.resid) in s1_frame_results_dict:
-                                        s1_frame_results_dict[(a.resname, a.resid)] = []
-                                    s1_frame_results_dict[(a.resname, a.resid)].append((h.index, a.index,
+                                    if not (a.resname, a.resid) in \
+                                    s1_frame_results_dict:
+                                        s1_frame_results_dict[(a.resname,
+                                        a.resid)] = []
+                                    s1_frame_results_dict[(a.resname,
```

Comment:
format as 
```python
s1_frame_results_dict[(a.resname, a.resid)].append(
        (h.index, a.index,
         (h.resname, h.resid, h.name),
         (a.resname, a.resid, a.name),
         dist, angle))
```

Reply:
Please indent for legibility. Follow [PEP8 indentation rules](https://www.python.org/dev/peps/pep-0008/#indentation)